<?xml version="1.0"?>
<launch>
    <node name="teleopduck1" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" output="screen">
    <remap from="/cmd_vel" to="/cmd_vel/duck1"/>
    </node>
</launch>
